.gitignore

```diff
@@ -23,3 +23,4 @@ testbin/*
 *.swp
 *.swo
 *~
+.vscode/launch.json
```

Comment:
Why is this line added ?

Reply:
For debugging with vscode, we need to add launch.json file in the root directory of the branch, added the file to gitignore, so that launch.json file is not added as new file in the repo. Changes to files in this list are ignored by GIT.